pallets/attestation/src/tests/delete.rs

```diff
@@ -37,57 +37,9 @@ fn test_remove() {
 			assert_ok!(Attestation::remove(
 				DoubleOrigin(ACCOUNT_00, attester.clone()).into(),
 				claim_hash,
-				None
-			));
-			assert!(Attestation::attestations(claim_hash).is_none());
-			assert_eq!(
-				events(),
-				vec![
-					Event::AttestationRevoked {
-						attester: attester.clone(),
-						claim_hash,
-						authorized_by: attestation::authorized_by::AuthorizedBy::Attester(attester.clone()),
-						ctype_hash,
-					},
-					Event::AttestationRemoved {
-						attester: attester.clone(),
-						claim_hash,
-						authorized_by: attestation::authorized_by::AuthorizedBy::Attester(attester.clone()),
-						ctype_hash,
-					}
-				]
-			);
-		});
-}
-
-#[test]
-fn test_remove_revoked() {
-	let attester: AttesterOf<Test> = sr25519_did_from_public_key(&ALICE_SEED);
-	let claim_hash = claim_hash_from_seed(CLAIM_HASH_SEED_01);
-	let mut attestation = generate_base_attestation::<Test>(attester.clone(), ACCOUNT_00);
-	attestation.revoked = true;
-	let ctype_hash = attestation.ctype_hash;
-
-	ExtBuilder::default()
-		.with_balances(vec![(ACCOUNT_00, <Test as Config>::Deposit::get() * 100)])
-		.with_ctypes(vec![(attestation.ctype_hash, attester.clone())])
-		.with_attestations(vec![(claim_hash, attestation)])
-		.build_and_execute_with_sanity_tests(|| {
-			assert_ok!(Attestation::remove(
-				DoubleOrigin(ACCOUNT_00, attester.clone()).into(),
-				claim_hash,
-				None
+				authorization_info
```

Comment:
Release 1.14.4 contains the broken attestation events?

Reply:
Ah, wait 1.14.4 only contained fixes for the node right?

Reply:
This looks like a GitHub bug. The two files are exactly the same if you open them on each respective branch. They also don't show as diff in my VS Code. Uh, that makes the whole point of this review kinda worthless. Maybe I can try to adjust the history, let's see.

Reply:
I would suggest you look at the PR from VS Code, which seems to be better at showing actual diffs.